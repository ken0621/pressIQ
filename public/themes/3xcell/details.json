{
	"theme_name":"3xcell Marketing",
	"description":"",
	"colors":
	{
		"default" : ""
	},
	"exclusive":"3xcell"
}