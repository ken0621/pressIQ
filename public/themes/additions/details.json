{
	"theme_name":"Additions PH",
	"description":"",
	"colors":
	{
		"default" : ""
	},
	"exclusive":"additions"
}