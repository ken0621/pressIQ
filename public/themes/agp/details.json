{
	"theme_name":"Alpha Global Prestige",
	"description":"",
	"colors":
	{
		"white" : ""
	},
	"exclusive":"agp"
}