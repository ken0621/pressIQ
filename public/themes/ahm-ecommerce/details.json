{
	"theme_name":"AHM Ecommerce",
	"description":"",
	"colors":
	{
		"default" : ""
	},
	"exclusive":"ahmecommerce"
}