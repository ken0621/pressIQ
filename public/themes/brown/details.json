{
	"theme_name":"Brown (MyPhone)",
	"description":"",
	"colors":
	{
		"default" : ""
	},
	"exclusive":"myphone"
}