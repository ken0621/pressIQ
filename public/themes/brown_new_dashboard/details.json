{
	"theme_name":"Brown New Dashboard",
	"description":"",
	"colors":
	{
		"default" : ""
	},
	"exclusive":"myphone"
}