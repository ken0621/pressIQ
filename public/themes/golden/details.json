{
	"theme_name":"Golden Falcon Theme",
	"description":"This page is noob",
	"colors":
	{
		"brown" : "",
		"gray" : "",
		"red" : ""
	}
}